.github/workflows/npm-cross-chain-arbitrum.yml

```diff
@@ -0,0 +1,58 @@
+name: NPM cross chain Arbitrum contracts
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "cross-chain/arbitrum/**"
+      - "cross-chain/arbitrum/deploy/**"
+      - "cross-chain/arbitrum/hardhat.config.ts"
+      - "cross-chain/arbitrum/package.json"
+      - "cross-chain/arbitrum/yarn.lock"
```

Comment:
Isn't it enough to just have `"cross-chain/arbitrum/**"`? That looks for all files in `arbitrum` directory recursively.

Alternatively, if we want to keep it similar to https://github.com/keep-network/tbtc-v2/blob/main/.github/workflows/npm-contracts.yml and list the paths explicitly, we can get rid of `"cross-chain/arbitrum/**"` and use `"cross-chain/arbitrum/contracts/**"` instead

Reply:
I looked at `solidity` package and we have `npm-contracts.yml` and `contracts.yml` and I followed the pattern thinking there might be a reason. However, after discussing this approach with Michalina after your comment, we decided to give it a shot and merge these two files. Now we have only one file for CI arbitrum workflow `cross-chain-arbitrum.yml`

Reply:
Our typical approach was to have 2 separate workflows - one to do testing (lint, format, build, run tests etc) and publish the testnet package, the other to publish the package for development purposes. Now that you asked the question, I have tried to remember why we were splitting those workflows and couldn't find a good argument for that :) The splitting could make sense if we were using conflicting workflow triggers. Maybe in the past it made sense and we were replicating some design patterns, without noticing it does no longer make sense...
Anyway, I've discussed the matter with Dmitry and we came up with the way to connect the workflows into one file.
The only significant change in how the things will run now is that in order to manually trigger the publishing of the package for development purposes (which may be needed in some rare situations, like after GH outage which blocked publishing of the package on push to main) we will need to provide `dev` as an `environment` input (rest of the inputs will be ignored).